trustchain-cli/src/bin/main.rs

```diff
@@ -72,17 +78,42 @@ fn cli() -> Command {
                         .about("Signs a credential.")
                         .arg(arg!(-v - -verbose).action(ArgAction::SetTrue))
                         .arg(arg!(-d --did <DID>).required(true))
+                        // TODO: credential file Should be required?
                         .arg(arg!(-f --credential_file <CREDENTIAL_FILE>).required(false))
```

Comment:
Yes, adding suggestion:
```suggestion
                        .arg(arg!(-f --credential_file <CREDENTIAL_FILE>).required(true))
```